channeldb/invoices.go

```diff
@@ -149,9 +191,13 @@ func (d *DB) AddInvoice(i *Invoice) error {
 			return err
 		}
 
+		paymentHash, err := getPaymentHash(i)
```

Comment:
Did you see any good reason for why `err` is in the second position for these returned values?

Reply:
Hah oh jeez (my mind is blown)